File: 1539-kth-missing-positive-number/1539-kth-missing-positive-number.py
class Solution(object):
    def findKthPositive(self, arr, k):
        """
        :type arr: List[int]
        :type k: int
        :rtype: int
        """
        j = 0 
        cnt = 1
        for i in range(1,2001,1):
            if j<len(arr) and arr[j]==i:
                j=j+1
            elif cnt==k:
                return i
            else:
                cnt+=1
        return i
            
        

File: 1539-kth-missing-positive-number/1539-kth-missing-positive-number.cpp
class Solution {
public:
    int findKthPositive(vector<int>& arr, int k) {
        int j=0,n=arr.size(),cnt=1,i;
        for( i=1;;i++)
        {
            if(j<n && arr[j]==i) 
                j++;
            else if(cnt==k) return i;
            else cnt++;
        }
        return i;
    }
};